src/index.ts

```diff
@@ -21,19 +21,17 @@ export interface InstructionMap {
   [key: string]: (element: Element) => void;
 }
 
-/**
- * A collection of tagName to errors.
- */
-type ErrorMultimap = Map<string, string[]>;
-
-export type Result<T> =
-  | {type: 'success'; value: T}
-  | {type: 'failure'; error: ErrorMultimap};
+function defaultHandleError(tagName, e: Error) {
+  const errorMsg =
+    `[bento-compiler]: Error thrown while rendering <${tagName}>\n` + e.stack;
```

Comment:
The wrapper text doesn't seem to add much in this context, since it'll be prefixed with `[JS] ` already and only in the Bento Compiler project anyway. WDYT about something closer to what we do in AMP with
```javascript
`[${tagName}] - ${errorMsg}`
```

Reply:
I'll throw some errors and see how they look in terminal output. That is how I got to the current incarnation

Reply:
**before**
![Screen Shot 2021-09-09 at 12 27 36 PM](https://user-images.githubusercontent.com/4656974/132725291-6d69d311-bd6f-4730-a3cc-d1529cdb3308.png)

**after**
![Screen Shot 2021-09-09 at 12 27 27 PM](https://user-images.githubusercontent.com/4656974/132725316-257b1239-72f1-4e7c-b00d-c6fad0e9b0e2.png)


I'll do the change, seems fine